__device-tests__/pages/editor-page.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @flow
+ * @format
+ * */
+/**
+ * External dependencies
+ */
+import wd from 'wd';
+/**
+ * Internal dependencies
+ */
+import BlockInteraction from '../blocks/block-interaction';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export default class EditorPage {
```

Comment:
Not sure what "Page" means in this context. Are you perhaps referring to a "screen" in the sense of multiple screens changing as the app moves from one to the other?

I think i'm a bit confused because "page" has special meaning within WordPress itself.

Reply:
Hmm it really is just referencing the only page in the Sample app with the Gutenberg Editor, figured it EditorPage was a good name but there can be some confusion based on the references within WordPress itself... Do you think we need to change this?

Reply:
Cool, I see, thanks! I'd say there's no need to change the name 👍